recipes/gn/all/conandata.yml

```diff
@@ -0,0 +1,3 @@
+sources:
+  "cci.20210429":
+    url: "https://gn.googlesource.com/gn/+archive/6771ce569fb4803dad7a427aa2e2c23e960b917e.tar.gz"
```

Comment:
Why no sha256?

Reply:
It changes all the time.
Some time stamp is probably spoiling the *fun*.

Reply:
yep, their servers produce tarballs on fly, and sha256 is always different, it's pain to package. if someone can help how to get stable tarball URL for gn, help is welcome.

Reply:
What about creating a mirror in Github that syncs itself automatically using actions?
It seems possible https://github.com/isaacs/github/issues/415#issuecomment-769711216

Reply:
I'm wondering if we could mirror this repo on github and have it sync itself automatically using actions.

Reply:
Something like this https://github.com/actions/checkout/issues/24
But it looks like it only works to sync repos on the same domain.

Reply:
It's completely possible.
But somebody/organization should invest some time setting it up...